covid19-etl/etl/dsfsi.py

```diff
@@ -0,0 +1,418 @@
+import copy
+import csv
+import datetime
+import re
+from contextlib import closing
+
+import requests
+
+from etl import base
+from helper.metadata_helper import MetadataHelper
+
+
+def format_subject_submitter_id(country, submitter_id):
+    submitter_id = "subject_dsfsi_{}_{}".format(country.lower(), submitter_id)
+    submitter_id = re.sub("[^a-z0-9-_]+", "-", submitter_id)
+    return submitter_id
+
+
+def format_demographic_submitter_id(subject_submitter_id):
+    return subject_submitter_id.replace("subject_", "demographic_")
+
+
+def normalize_current_status(status):
+    normalized = {
+        "?": None,
+        "Alive": "alive",
+        "Clinically Stable": "stable",
+        "critical condition": "critical",
+        "Critical condition": "critical",
+        "Dead": "deceased",
+        "Death": "deceased",
+        "Deceased": "deceased",
+        "deceased": "deceased",
+        "Died": "deceased",
+        "In recovery": "in recovery",
+        "In treatment": "in treatment",
+        "NA": None,
+        "Pos": "positive",
+        "Receiving Treatment": "in treatment",
+        "Recovered": "recovered",
+        "recovered": "recovered",
+        "Recovered (3/19/2020)": "recovered",
+        "Stable": "stable",
+        " Stable": "stable",
+        "stable condition": "stable",
+        "Tested Negative and hence is not considered as a case anymore": None,
+        "Under treatment": "in treatment",
+        "unstable": "unstable",
+    }
+
+    return normalized[status]
+
+
+def normalize_symptoms(symptoms):
+    normalized = {
+        "acute pneumonia": "pneumonia",
+        "Asymptomatic": "asymptomatic",
+        "breathing complications": "trouble breathing",
+        "chest pain": "persistent pain or pressure in the chest",
+        "cold": "fever* or feeling feverish/chills",
+        "Cold": "fever* or feeling feverish/chills",
+        "Cough Fever": "cough",
+        "cough": "cough",
+        "Cough": "cough",
```

Comment:
you could do a `.lower` on the key instead of duplicating Cough, Cold, Fever etc